src/envs/cover.py

```diff
@@ -631,7 +655,7 @@ def _create_initial_state(self, rng: np.random.Generator) -> State:
             # [is_block, is_target, width, x]
             data[target] = np.array([0.0, 1.0, width, x])
         # [x, y, grip, holding]
-        data[self._robot] = np.array([0.0, self.initial_robot_y, 0.0, -1.0])
+        data[self._robot] = np.array([0.0, self.initial_robot_y, -0.1, -1.0])
```

Comment:
If the grip value is greater than 0, the gripper is in the ON position. If it is less than or equal to 0, the gripper is in the OFF position. I thought it's less confusing if the value we set is not the threshold value, since someone may not remember if the logic is is > or >= to set it to ON. 

I now changed it to +1.0 and -1.0 instead of +/- 0.1, as 0.1 is more arbitrary than +/- 1.0